xml/System.IO.Pipes/NamedPipeServerStreamAcl.xml

```diff
@@ -77,11 +77,11 @@ If `options` contains &lt;xref:System.IO.Pipes.PipeOptions.CurrentUserOnly&gt;,
 
 -or-
 
-<paramref name="inBufferSize" /> is a negative number.
+<paramref name="inBufferSize" /> or <paramref name="outBufferSize" /> is a negative number.
```

Comment:
```suggestion
<paramref name="inBufferSize" /> or <paramref name="outBufferSize" /> is less than zero.
```
For consistency with https://github.com/dotnet/dotnet-api-docs/blob/61a1524e55f4c326216ace4007f97881f5d04954/xml/System.IO.Pipes/AnonymousPipeServerStreamAcl.xml#L65

Reply:
Sure. Also, you caught a bug: I merged the latest suggestion with a duplicated closing `</exception>` tag.